src/magic_folder/magicfolderdb.py

```diff
@@ -24,6 +24,10 @@
     ActionType,
 )
 
+from functools import (
+    wraps
```

Comment:
Personally, I like trailing commas here since it minimizes future diffs.  Right now this is purely a style preference so change it only if you want.  (I might try to formalize formatting as some kind of [black configuration](https://pypi.org/project/black/) and make it a CI check someday though.)